build.rs

```diff
@@ -3,6 +3,5 @@
 // SPDX-License-Identifier: Apache-2.0
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tonic_build::compile_protos("proto/manager.proto")?;
```

Comment:
It seems the file can delete？None  use in sealfs.

Reply:
https://github.com/labring/sealfs/blob/main/proto/manager.proto can also delete.

Reply:
https://github.com/labring/sealfs/blob/main/proto/fsbase.proto seems useless?I'm not sure

Reply:
`build.rs` and `manager.proto` have been deleted, but `fsbase.proto` does not know whether to delete

Reply:
@uran0sH @luanshaotong Is https://github.com/labring/sealfs/blob/main/proto/fsbase.proto useless？

Reply:
> `build.rs` and `manager.proto` have been deleted, but `fsbase.proto` does not know whether to delete

I think it is useless. Pls remove it.

Reply:
By the way，tonic dependency can delete

Reply:
> By the way，tonic dependency can delete

Yes!

Reply:
All about `tonic` deleted